src/requestor/routes/manage.py

```diff
@@ -89,9 +91,39 @@ async def create_request(
         f"Creating request. request_id: {request_id}. Received body: {data}. Revoke: {'revoke' in api_request.query_params}"
     )
 
-    # error (if we have both resource_path and policy_id) OR (if we have neither)
-    if bool(data.get("resource_path")) == bool(data.get("policy_id")):
-        msg = f"The request must have either a resource_path or a policy_id."
+    if data.get("resource_path"):
+        if not data.get("resource_paths"):
+            data["resource_paths"] = [data["resource_path"]]
+        else:
+            # error if we have both resource_paths and resource_path
+            msg = f"The request must have either resource_paths or a resource_path."
+            logger.error(
+                msg + f" body: {body}",
+                exc_info=True,
+            )
+            raise HTTPException(
+                HTTP_400_BAD_REQUEST,
+                msg,
+            )
+
+    # error (if we have both policy_id and (resource_paths or resource_path))
+    # OR (if we have neither)
+    if bool(data.get("policy_id")) == (
+        bool(data.get("resource_paths")) or bool(data.get("resource_path"))
+    ):
+        msg = f"The request must have either resource_paths or a policy_id."
+        logger.error(
+            msg + f" body: {body}",
+            exc_info=True,
+        )
+        raise HTTPException(
+            HTTP_400_BAD_REQUEST,
```

Comment:
don't think this is a big issue, but should we raise 400 in this case? or just simply let `resource_paths` to take precedence over `resource_path`?

Reply:
Yes, good idea! I've added a commit where `resource_paths` takes precedence and have removed the error when both are present.